src/main/java/net/sf/jabref/logic/mods/MODSEntry.java

```diff
@@ -116,11 +116,11 @@ private void populateFromBibtex(BibEntry bibtex) {
         if (bibtex.hasField(BibEntry.KEY_FIELD)) {
             id = bibtex.getCiteKey();
         }
-        if (bibtex.hasField("place")) { // TODO: "place" is the MODS version, in BibTeX: "address", BibLaTeX: "location"?
+        if (bibtex.hasField("address")) { // TODO: "place" is the MODS version, in BibTeX: "address", BibLaTeX: "location"?
```

Comment:
Have a look at the getFieldAlias (just don't renenber the correct name).  It allows you to get the value .e.g adress or journal, independent of the actual field. For example when year not present than the value is extracted from the date field